services/QuillLMS/client/app/bundles/Staff/components/evidence/syntheticData/seedDataForm.tsx

```diff
@@ -18,28 +18,83 @@ const SeedDataForm = ({ history, match }) => {
 
   const [activityNouns, setActivityNouns] = React.useState<string>('');
 
+  const blankLabelConfig = { label: '', examples: ['',''], };
+  const blankLabelConfigs = { [BECAUSE] : [], [BUT] : [], [SO] : [], };
+
+  const [labelConfigs, setLabelConfigs] = React.useState({...blankLabelConfigs});
+
+  function handleLabelConfigsChange(event, index, conjunction, key) {
+    const data = labelConfigs
+    const conjunctionData = [...data[conjunction]];
+
+    conjunctionData[index][key] = event.target.value;
+    data[conjunction] = conjunctionData;
+
+    setLabelConfigs(labelConfigs => ({...data}));
```

Comment:
This works fine, but since you aren't ever using the `labelConfigs` argument here and don't really need to worry about mutating `data` since it's only defined in the function scope, the more succinct way to write this would just be `setLabelConfigs(data)`. This is true below as well.

Reply:
Good note. I've removed the `labelConfigs` to make it `setLabelConfigs({...data})`. Note, I learned this during this project, but since the underlying `useState` item here is an object with properties I'm updating, i.e. a `{'because' : [], 'but': [], 'so' []}` object in which I edit the arrays , it seems I need to clone it (i.e. `{...data}` vs. `data`) to make the `state` update properly (this surprised me). I believe this is because of the underlying diff logic will otherwise see it as the same object and not update (if you don't clone the object, the code doesn't function).

And to just make this cleaner, I'll move the cloning up to the `data` assignment
```javascript
const data = {...labelConfigs}
setLabelConfigs(data)